src/styles/utils/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  width: 706px;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  /* I must to put margin 122px and 72px for passing npm test without them it's not works */
+  &--position {
+    margin: 72px auto;
+  }
+
+  &--start-day-wed,
+  &--month-length-29 {
+    margin: 122px auto;
+  }
+}
```

Comment:
just center the calendar instead of hardcoding the margin

Reply:
great, thank you, i fixed it